api.go

```diff
@@ -97,7 +97,7 @@ func DeletePod(podID string) (VCPod, error) {
 		return nil, errNeedPodID
 	}
 
-	lockFile, err := lockPod(podID)
+	lockFile, err := lockPod(podID, true)
```

Comment:
just a nit: It would be nice if the exclusive arg gave some visual clue to if it was an exclusive or not request, something like:
`lockPod(podID, LOCK_EXCL)`
or similar - just to make it clearer when reading the code what the bool arg is saying. But, I don't think there is any easy existing arg type we can purloin for that. If it's easy to add a type just for this, maybe consider it.

Reply:
Yes I can introduce `exclusivePodLock` and `sharedPodLock`.